configure.ac

```diff
@@ -527,7 +527,6 @@ AC_SUBST([ZLIB_CPPFLAGS])
 AC_SUBST([ZLIB_LDFLAGS])
 AC_SUBST([ZLIB_LIBS])
 
-
```

Comment:
Perhaps add that line back to reduce the gratuitous diff?

Reply:
will do

Reply:
cf 4df77b2